### PR TITLE
Textual improvements

### PR DESCRIPTION
**Pull request by Jay-o-Way**
Grammar, spelling, formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+
+All letters are sorted by usage frequency by default, as found on [Wikipedia](https://en.wikipedia.org/wiki/Letter_frequency).
 
 # Download
-See the [Release](https://github.com/leroyd/PowerAccent/releases) page
 
-# Problem known
-- Some keys can have weird behavior with some actions or some tools
-- (Experimental feature) Inside browsers & some other softwares, the tool can't detect the caret position. So default is applied, meaning up center of the main window
+See the [Release](https://github.com/leroyd/PowerAccent/releases) page.
```
Comment (Jay-o-Way) [inline README.md:11]: Are you using two repositories for the same project?

Comment (damienleroy) [reply README.md:11]: haha, no. I changed the name of the repo since started the project.

Comment (Jay-o-Way) [reply README.md:11]: Sorry, I didn't mean the _repo_ name, I meant the `leroyd` part, but I think it auto-forwards to the correct account. Still better to use current name in docs.

Comment (damienleroy) [reply README.md:11]: Yeah sorry, I would say the account name. Changed it from `leroyd` to `damienleroy`.

**Pull request merged**